charts/token-exchange-api/values.yaml

```
@@ -34,6 +34,7 @@ env:
   IDENTITY_URL: http://identity-api-dev:8080/query
   IPFS_BASE_URL: https://assets.dimo.xyz/ipfs
   DEV_LICENSE_VALIDATOR_FEATURE_FLAG: true
+  IPFS_TIMEOUT: 30
```

This can be a string that we call [time.ParseDuration](https://pkg.go.dev/time#ParseDuration) on
```suggestion
  IPFS_TIMEOUT: 30s
```

---

Would be nice if our config loader understood `time.Duration`!